docs/reference/runtime-fields.asciidoc

```diff
@@ -1,15 +1,17 @@
-[[alerts-runtime-fields]]
-== Add runtime fields to detection alerts
+[[runtime-fields]]
+== Create runtime fields in {elastic-sec}
 
 Runtime fields are fields that you can add to documents after you've ingested your data. For example, you could combine two fields and treat them as one, or perform calculations on existing data and use the result as a separate field. Runtime fields are evaluated when a query is run. 
 
-You can add a runtime field to your detection alerts from the *Alerts* table. The new field applies to all {elastic-sec} alerts.
+You can create a runtime field and add it to your detection alerts or events from any page that lists alerts or events in a data grid table, such as *Alerts*, *Timelines*, *Hosts*, and *Users*. The new field becomes available to all {elastic-sec} alerts and events.
 
 NOTE: Runtime fields can impact performance, because they're evaluated each time a query runs. Refer to {ref}/runtime.html[Runtime fields] for more information.
 
-To add a runtime field to detection alerts:
+To create a runtime field:
 
-. Go to *Detect* -> *Alerts*, then click the *Fields* toolbar button at the upper-left of the *Alerts* table. The *Fields* browser opens.
+. Go to a page that lists alerts or events (for example, *Detect* -> *Alerts* or *Investigate* -> *Timelines* -> *_Name of Timeline_*).
+
+. Click the *Fields* toolbar button at the upper-left of the table. The *Fields* browser opens.
```

Comment:
I removed "toolbar" because I don't think we normally reference that UI element in data grid tables. Though, maybe we _should_ start including that element when orienting users towards certain buttons above tables? Toolbars are common in modern web design and we have them in most of our data grid tables that have sorting/config options. Including it could lead to more verbose writing though. I'm also not convinced that our tables are so complex that users need extra help orienting themselves on the page or finding certain buttons along the top of the table. WDYT?

Reply:
@nastasha-solomon I agree about "toolbar"; I don't see a strong need to include it, especially since the sentence also provides directional guidance on finding the button. But I don't see significant value in "_located_ at the upper-left" vs. "at the upper-left", as they're both clear directions. What was your thinking on that? IMO it feels kind of like "in order to" vs. "to."

Reply:
Ah, I see what you're saying and agree that it's not necessary. I guess I was expecting a verb after the comma when I read the sentence out loud.

Reply:
@nastasha-solomon Yeah the comma changes things. I'd go without a comma, just omit "toolbar":
```suggestion
. Click the *Fields* button at the upper-left of the table. The *Fields* browser opens.
```

Reply:
Adding to the mix here. Small question -- "at the upper-left of the table" sounds a bit odd to me. Is `in the upper-left` or `in the table's upper-left` more grammatically correct since the button is technically inside the table?

Look at us pondering over these nits. 😆 I enjoy reading y'alls thoughts!

Reply:
@jmikell821 I agree, the prepositional phrases in the original sound a little overly constructed. I think all those options are grammatically correct, but "in the table's upper-left" flows better and is most specific about the location.
```suggestion
. Click the *Fields* toolbar button in the table's upper-left. The *Fields* browser opens.
```